jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildDockerMojo.java

```diff
@@ -85,6 +91,18 @@ public void execute() throws MojoExecutionException {
             .setEnvironment(getEnvironment())
             .setExposedPorts(ExposedPortsParser.parse(getExposedPorts()))
             .setAllowHttp(getAllowInsecureRegistries());
+    if (Files.exists(getExtraDirectory())) {
```

Comment:
if they explicitly specify a directory that doesn't exist, should we say so?

Reply:
Yes, I'll add a TODO for now that should be resolved when this feature is more solidified.